source/_components/switch.pilight.markdown

```diff
@@ -55,6 +55,12 @@ Variables for the different codes:
 - **'off'** (*Optional*): `1` or `0`
 - **'on'** (*Optional*): `1` or `0`
 
+Besides that the he `on_code_receive` and `off_code_receive` have the following special option:
```

Comment:
Please cleanup "Besides that the he". Also I think this would be less verbose/clearer as a linkable section heading e.g. "`on_code_receive`/`off_code_receive` options" with a link from "echo" documented under both on_code_receive/off_code_receive's variable section above (lines 45+46).